transforms/helpers/transform-helper.ts

```diff
@@ -301,45 +302,40 @@ function createActionDecoratedProps(
 function createCallExpressionProp(
   j: JSCodeshift,
   callExprProp: EOCallExpressionProp
-): MethodDefinition[] | ClassProperty[] {
+): ClassMethod[] | ClassProperty[] {
```

Comment:
This whole file looks good. If you liked the changes I suggested in `ast.ts` (around making type
aliases for unions), it makes sense to do the same thing here.

But since *I'm* already getting tired at this point, feel free to punt.